### PR TITLE
Microverse Portfolio: Added headline section

### PR DESCRIPTION
**Pull request by Graycemuthui**
Created a repository on GitHub.
Added linter configurations.
Created .gitignore file.
Created an image folder with all the project images.
Created HTML file and added a header with a heading and a menu icon.
Created a CSS file and added style to the header element.
Created a headline section with social links.
Added style to the headline section
Used proper GitHub flow.
Followed all project and general guidelines.
## Changes made
Wrapped icons in the list.
Updated README.md file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+              class="headline-icon"
+          /></a>
+          <a href=""
+            ><img
+              src="./images/m-logo.svg"
+              alt="A m-logo"
+              class="headline-icon"
+          /></a>
+          <a href=""
+            ><img
```
Comment (phelian23) [inline index.html]: - [ ] Good job with your project so far. However, for semantic purposes try to wrap the list of icons in a `<ul>` tag and each one should be `<li>` tag as improve your SEO benchmark and your section will be structured
example :
``` markup
<ul>
  <li>facebook icon</li>
  <li>twitter icon</li>
</ul>
```

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @Graycemuthui, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ Linters are passing
✅ Descriptive pull request
✅ Good commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  line-height: 1.1;
+  width: 80%;
+}
+
+.headline-text {
+  color: var(--color-text);
+  line-height: 1.1;
+  margin-top: 2rem;
+  padding-right: 1.5rem;
+}
```
Comment (vic778) [inline css/styles.css:91]: - OPTIONAL[] kindly add this `font-size: 20px;
line-height: 32px;` to make your paragraph readable.

**Review by vic778 — APPROVED** [review]
Hi @Graycemuthui  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Charlie Brown

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**